### DISCUSSION TITLE
GHelper on AtlasOS

### DISCUSSION THREAD

**Discussion by Samer1FL** [seerge/g-helper#403] [Q&A]
Hello, i’m using AtlasOS and i’m just wondering what are the services that Ghelper uses to adjust fans, i’m an X13 XGM user

**Comment by seerge** [comment]
G-helper doesn't use any windows services to adjust fans.

**Reply by Samer1FL** [reply]
Thank you Sergee, such an amazing App, however I strongly recommend you to refer back to Armory Control to know how XGM runs. I love Ghelper to control my CPU fan boost, but it is missing XGM. Armory Control has control over XGM, but it has no control such as Ghelper over CPU fan boost. This feature is giving more power to XGM by disabling it. I’m glad to assist in this project by any kind since i own AMD XGM

**Reply by seerge** [reply]
@Samer1FL yes, i plan to eventually add that. Your topic is a dublicate of https://github.com/seerge/g-helper/issues/301
